src/sentry/models/savedsearch.py

```diff
@@ -13,6 +13,7 @@
     {"name": "Assigned To Me", "query": "is:unresolved assigned:me"},
     {"name": "My Bookmarks", "query": "is:unresolved bookmarks:me"},
     {"name": "New Today", "query": "is:unresolved age:-24h"},
+    {"name": "Unhandled Errors", "query": "is:unresolved error.unhandled:true"},
```

Comment:
I don't *think* we need this anymore if we're making this a global search. Unless we still have project level saved searches somewhere?

Reply:
I'm adding it for completeness sake. I'm also going to see if we can remove the project level saved searches entirely so this code can leave then.